mettle/src/stdapi/sys/process.c

```diff
@@ -27,8 +28,32 @@ get_process_info(sigar_t *sigar, sigar_pid_t pid)
 
 	p = tlv_packet_add_u32(p, TLV_TYPE_PID, pid);
 	p = tlv_packet_add_u32(p, TLV_TYPE_PARENT_PID, pstate.ppid);
-	p = tlv_packet_add_str(p, TLV_TYPE_PROCESS_NAME,
+
+	if (strcasestr(BUILD_TUPLE, "linux")) {
```

Comment:
This doesn't seem like the best way to determine if the current platform is Linux or not, but it was the only thing I could find without performing fingerprinting myself. If there's a better way, please let me know! The `HOST_LINUX` and `HOST_LINUX_TRUE` are available for automake don't appear to be exposed as C preprocessor macros.